ivr/src/main/java/org/motechproject/ivr/service/impl/OutboundCallServiceImpl.java

```diff
@@ -102,6 +102,12 @@ private void addCallDetailRecord(String callStatus, Config config, Map<String, S
     }
 
     @Override
+    public void initiateCall(Map<String, String> params) {
+        String defaultConfig = configService.getDefaultConfig();
+        initiateCall(defaultConfig, params);
```

Comment:
add throw new ConfigNotFoundException when default config is blank or null. Also add information about exception to the javadoc